mlibrary/catalog-browse
# [SEARCH-1558] Add an alert about browse's beta status

# Overview
The `m-website-banner` that tells users about the site being in beta turns out to be not very affective. This will be replaced by the Design System's `m-callout` component.

This pull request closes [SEARCH-1558](https://tools.lib.umich.edu/jira/browse/SEARCH-1558).

## Anything else?
### Styles
While adding styles for `m-callout`, stylesheets existed in two directories: `./css/styles` and `./public/styles`. The styles were being compiled from the `./css/styles` directory, so the `./public/styles` directory has been deleted.

### Error Message
The [error message for invalid inputs](https://www.figma.com/file/ymqm7jMhh1xFIONkNZZrcB/?node-id=381%3A9747) has been changed, so it has been updated.

### Browse List Specs
`./spec/models/browse_list_spec.rb` did not include testing for `Empty` and `Error`, so those tests have been added.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go to [the site](http://localhost:4567/callnumber) and see the message.
  - If you click `Dismiss`, does it go away?
  - Search for a call number, do you see it appear?
  - If you navigate by clicking `Previous page` or `Next page`, does the callout not appear?
